config/environments/test.rb

```diff
@@ -1,6 +1,9 @@
 PrisonVisits2::Application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
 
+  ENV['SMTP_USERNAME'] = 'Test'
+  ENV['SMTP_PASSWORD'] = '12345678'
```

Comment:
We probably shouldn't be directly setting (or using, as I mention below) environment variables. See if you can set `config.action_mailer.smtp_settings.user_name` and `config.action_mailer.smtp_settings.password`, maybe?